src/dmd/hdrgen.d

```diff
@@ -3609,13 +3609,19 @@ private void initializerToBuffer(Initializer inx, OutBuffer* buf, HdrGenState* h
         ei.exp.expressionToBuffer(buf, hgs);
     }
 
+    void visitC(CInitializer ci)
+    {
+        buf.writestring("hdrgen C initializers TODO");
```

Comment:
Would it be too much to deal with this now?

Reply:
yes, the PR is big enough

Reply:
https://github.com/dlang/dmd/pull/12644